src/gsi/crtm_interface.f90

```diff
@@ -891,14 +915,14 @@ subroutine destroy_crtm
   error_status = crtm_destroy(channelinfo)
   if (error_status /= success) &
      write(6,*)myname_,':  ***ERROR*** error_status=',error_status
-  if (n_actual_aerosols_wk>0 .or. n_clouds_fwd_wk>0) then  
-     if (imp_physics==11) then
```

Comment:
why is this physics if removed?

Reply:
This is what is in the current EMC GSI master. 
My EMC GSI master base is on March 8 2022. I copied this subroutine from EMC master when I resolved the conflicts. It turned out that the subroutine is a little advanced than the one in my gsi base.